README.md

```diff
@@ -1,11 +1,15 @@
-# LasIO
+|                          **Build Status**                          |
+|:------------------------------------------------------------------:|
+| [![Actions Status](https://github.com/msbahal/LasIO.jl/workflows/Linux/badge.svg)](https://github.com/msbahal/LasIO.jl/actions) [![Actions Status](https://github.com/msbahal/LasIO.jl/workflows/Windows/badge.svg)](https://github.com/msbahal/LasIO.jl/actions) |
```

Comment:
I'm ok with this if we stick to this fork. But this needs to change back if we want to attempt to merge this to the orignal LasIO...

Reply:
Yes that is the plan. However, we might be able to convince the guys to use GitHub actions instead of Travis...